main.py

```diff
@@ -232,9 +274,10 @@ def escape_path(path):
 		accelerator_arg = Xqemu.generateAcceleratorArg(settings.settings['use_accelerator'])
 
 		dvd_path_arg = ''
-		if settings.settings['dvd_present']:
-			check_path(settings.settings['dvd_path'])
-			dvd_path_arg = ',file=' + escape_path(settings.settings['dvd_path'])
+		if mwindow.tableGames.currentRow() > 0:
+			selectedGame = settings.settings['iso_path'] + '/' + mwindow.tableGames.item(mwindow.tableGames.currentRow(), 0).text().split('\n')[1]
```

Comment:
The filename should not be extracted from the UI element.
We should store extra-data per element, so the UI and logic are entirely seperated.